apollo-router/src/plugins/connectors/tests.rs

```diff
@@ -565,22 +609,31 @@ async fn execute(
     }));
 
     let config_object = config.as_object_mut().unwrap();
-    config_object.insert(
-        "preview_connectors".to_string(),
-        serde_json::json!({
-          "subgraphs": {
-            "connectors": {
-              "json": {
-                "override_url": connector_uri
-              }
-            }
-          }
-        }),
-    );
+    config_object
```

Comment:
is there a deep_merge fn or something similar this easier to read?

Reply:
Not built in, but I found a crate which does it and added it.